profiler/src/profile_max_pool3d_fwd.cpp

```diff
@@ -118,7 +118,9 @@ int profile_max_pool3d_fwd(int argc, char* argv[])
 #ifdef CK_ENABLE_FP32
     using F32 = float;
 #endif
+    using I8    = int8_t;
     using I32   = int32_t;
+    using F8    = ck::f8_t;
     using NDHWC = ck::tensor_layout::convolution::NDHWC;
 
 #if 1
```

Comment:
Either remove this `#if 1` and it's `#else` part or add cmdline arg for reduce op id.